client/src/components/BoardModal/BoardModal.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+import Modal from '@mui/material/Modal';
+import { TextField } from '@mui/material';
```

Comment:
I recommend grouping these imports for clarity:
```suggestion
import { Box, Button, Modal, TextField, Typography } from '@mui/material';
```